### ISSUE TITLE
Ethereum Questions From A Newbie

### ISSUE THREAD

**Issue by BedoGesaratsi** [ethereum/mist#2104]
Hi friends, firstly thanks very much for your support and precious time.

1 - **"Accounts can't display incoming transactions, but hold and send ether. To see incoming transactions create a wallet contract to store ether."** - To make things clear, my default Ethereum Wallet is able to **hold / receive / send** ETHER, right? In other words, I don't need to create a wallet contract. Please anyone can clarify this?

2 - When creating my wallet for the very first time (let's assume the name of the Wallet is "**Jedi**"), the software forced me to create a password and I did it. But other than this, I was never asked to enter this password once again when I re-open my "**Jedi**" wallet from where it's installed. Is this normal? 

3 - When I open my wallet, I see my account's name and just below that I see an address, like "0x91jDıHkG........." this is the address I use to RECEIVE ether, right? So, it's "ok" if others see it (and they should see it), Please clarify.

4 - And finally, if there any, which security steps I may add to my Ethereum Wallet?

5 - For backing up (let's assume that my computer crashed, I built a new computer and want to get my wallet back), is it enough to SAVE the file in the KEYSTORE FOLDER?

Thanks for all your patience, I really appreciate it.

**Comment by sandakersmann** [comment]
Hi and welcome to the community :)

1 - Correct. You don't need to create a wallet contract to hold ETH.

2 - You need the password to spend funds.

3 - Correct. This is your public address.

4 - Yes. Run it on Linux. Many people have lost cryptocurrencies stored on a Windows system.

5 - Correct.

If you are going to store a larger amount I would recommend that you buy one of these for super secure storage: https://www.ledgerwallet.com/products/12-ledger-nano-s

This is also not really the place to ask these questions. I recommend that you ask here in the future:
https://www.reddit.com/r/ethereum and https://www.reddit.com/r/ethtrader

**Comment by BedoGesaratsi** [comment]
@sandakersmann  Thanks a lot, I really appreciate your help and patience.
Wish you the very best.
Bedo

**Comment by aristola** [comment]
Question regarding Mist 0.8.1

 I just downloaded Mist 0.8.1 and created a password. Then I transferred some ETH to the address provided. The blockchain was downloading so I clicked the "Back" and "Next" buttons, and suddenly I'm prompted to "Protect your account", i.e. create a password. How can I access the first address I made? All I have is the public key and the password I created. Can't seem to find a .json wallet file.
